src/AiClient/OllamaClient.ts

```diff
@@ -0,0 +1,45 @@
+	"stream": false
+}'
+*/
```

Comment:
ref: https://yc.prosetech.com/running-your-very-own-local-llm-6d4db99c0611

This will be removed once everything is working correctly.

Basic curl that worked locally was:

```bash
curl http://localhost:11434/api/generate -d '
{
 "model": "llama2",
 "prompt": "Would you like to talk?",
 "stream": false
}' | jq .response
```

Pasted in transcript fetched from https://www.youtube.com/watch?v=-LkvBMpCK-A to see if I could get it to run locally, but it is taking forever...